empty_workflow.py

```diff
@@ -0,0 +1,7 @@
+from kfp import dsl
+
+
+@dsl.pipeline(name="test_pipeline", description="empty pipeline")
+def pipeline():
+    dsl.ContainerOp(name="test", image="mlrun/mlrun", command=["Testing !"])
```

Comment:
does this run the containerop? I don't think its needed. you can use get or create context and log some message "running empty pipeline" and exit